translations/ui-invoice/en.json

```diff
@@ -387,6 +386,9 @@
   "invoice.actions.saveLine.error.budgetNotFoundByFundIdAndFiscalYearId": "Invoice line cannot be saved because Fund <b>{fundCode}</b> has no current budget.",
   "invoice.actions.saveLine.error.inactiveExpenseClass": "Invoice line can not be saved because expense class <b>{expenseClass}</b> is inactive.",
   "invoice.actions.saveLine.error": "Invoice line was not saved",
+  "invoice.actions.error.userHasNoPermission": "Operation is not permitted because user is not a member of the specified acquisition unit",
```

Comment:
Strange that two translations (and error codes respectively) have the same message

Reply:
Yes, BE effort will be needed to return the same error code. We had such cases in the receiving app as well.